presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/TransformQuantifiedComparisonApplyToScalarApply.java

```diff
@@ -164,7 +164,7 @@ private PlanNode rewriteQuantifiedApplyNode(ApplyNode node, QuantifiedComparison
                     node.getId(),
                     context.rewrite(node.getInput()),
                     subqueryPlan,
-                    Assignments.identity(minValue, maxValue),
+                    Assignments.identity(minValue, maxValue, countAllValue, countNonNullValue),
```

Comment:
it is not that this optimizer introduced that. It is rather a bug in previous optimizer.